src/store/index.ts

```diff
@@ -4,19 +4,21 @@ import actions from './actions'
 import getters from './getters'
 import mutations from './mutations'
 import { state } from './state'
+import CurrentAccount from '@/store/CurrentAccount'
```

Comment:
I was wondering where `CurrentAccount` came from since it wasn't declared anywhere. Is there any way to tie this back to the export in the index file?

Reply:
Vuex is pretty clever.  When referencing actions, getters or mutators you can call them directly without having to know the store in which they reside.  For example, both the following will work:
```
dispatch('getCurrentAccount')  // from the current account module
dispatch('someOtherAction') // from another module
```

We could choose to namespace the stores, but I don't think we'll have so many methods that we'll have name collisions.  Are you comfortable with that?

Reply:
No need to namespace the modules.

Resolving this comment.